pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -572,22 +575,22 @@ func (gb *GraphBuilder) addUnblockedOwnersToDeleteQueue(removed []metav1.OwnerRe
 		if wasBlocked && isUnblocked {
 			node, found := gb.uidToNode.Read(c.newRef.UID)
 			if !found {
-				klog.V(5).Infof("cannot find %s in uidToNode", c.newRef.UID)
+				logger.V(5).Info("cannot find uid in uidToNode", "uid", c.newRef.UID)
 				continue
 			}
 			gb.attemptToDelete.Add(node)
 		}
 	}
 }
 
-func (gb *GraphBuilder) processTransitions(oldObj interface{}, newAccessor metav1.Object, n *node) {
+func (gb *GraphBuilder) processTransitions(logger klog.Logger, oldObj interface{}, newAccessor metav1.Object, n *node) {
 	if startsWaitingForDependentsOrphaned(oldObj, newAccessor) {
-		klog.V(5).Infof("add %s to the attemptToOrphan", n.identity)
+		logger.V(5).Info("add node to attemptToOrphan", "node", n.identity)
 		gb.attemptToOrphan.Add(n)
 		return
 	}
 	if startsWaitingForDependentsDeleted(oldObj, newAccessor) {
-		klog.V(2).Infof("add %s to the attemptToDelete, because it's waiting for its dependents to be deleted", n.identity)
+		logger.V(2).Info("add node to attemptToDelete, because it's waiting for its dependents to be deleted", "node", n.identity)
```

Comment:
Should this be "item" to match the other calls?

Reply:
Updated